sig-architecture/production-readiness.md

```diff
@@ -59,16 +64,20 @@ aspects of the process.
     which it depends?
   - How does this feature respond to degraded performance or high error rates
     from services on which it depends?
+  - If this feature fails, what impact would it have on cluster workloads? 
 * Monitoring requirements
   - How can an operator determine if the feature is in use by workloads?
+     - Are these tests documented? 
```

Comment:
I think that answering the question constitutes the documentation, so we don't need to ask if they are documents. Same for the two below.

Reply:
removed those questions. thanks @johnbelamaric